catalog/files/api/src/main/java/org/projectnessie/catalog/files/api/BackendErrorCode.java

```diff
@@ -15,29 +15,29 @@
  */
 package org.projectnessie.catalog.files.api;
 
-import java.time.Instant;
-import java.util.Optional;
+public enum BackendErrorCode {
+  /**
+   * Treat unknown errors as non-retryable to prevent indefinite retry cycles when the server is not
+   * able to determine a specific failure reason. This may be changed in the future when task
+   * deadlines are supported.
+   */
+  UNKNOWN(false),
```

Comment:
Let's make `UNKNOWN` retryable - we don't know what happened, so it can be retryable.

Reply:
I know real `UNKNOWN` will end up in infinite retry

Reply:
I'm planning to add tast deadlines in a follow-up PR and also fix #8860